README.md

```diff
@@ -130,6 +130,43 @@ The [minikube start](./docs/minikube_start.md) command can be used to start your
 This command creates and configures a virtual machine that runs a single-node Kubernetes cluster.
 This command also configures your [kubectl](http://kubernetes.io/docs/user-guide/kubectl-overview/) installation to communicate with this cluster.
 
+### Configuring Kubernetes
+
+Minikube has a "configurator" feature that allows users to configure the Kubernetes components with arbitrary values.
+This feature currently supports the following components:
+
+* kubelet
```

Comment:
Maybe a reword to combine the supported component list with the documentation list at #L155-160 since they are the same list